### PR TITLE
Sm orderview template

### PR DESCRIPTION
**Pull request by shawn-mit**
# Description
This PR is for the template and html of Orderview. This template will allow a  Bangazonian access to select a payment or add a payment after completing an order. 

# Related Ticket(s)
List related ticket(s) that apply to the work done in this pull request
#12 
## Testing

[ x] There are new unit tests in this PR, and I verify that there is full coverage of all new code.
[x ] I certify that all existing tests pass

## Documentation

[x ] There is new documentation in this pull request that must be reviewed..

## Deploy Notes
Only the dependency files were pushed in this request.  

```sh
git pull --prune
git checkout <feature_branch>
```

## Steps to Test
Outline the steps to test the code in this pull request.

1. python manage.py runserver

### CODE REVIEW

**Comment by ben-d-marks** [comment]
This branch is several commits behind master, and hence has 2 conflicts. One is of the boilerplate order detail html, and the other is some corrections to the orderdetailview, which will change with the fix add to order PR.

File: bangazon/bangazon_ui/models/order_model.py [hunk 1]
```diff
@@ -17,6 +17,7 @@ class Order (models.Model):
     completed = models.IntegerField(default = 0)
     payment_type = models.ForeignKey(PaymentType, on_delete=models.CASCADE)
     product = models.ManyToManyField(Product)
+    #productorder= models.ManyToManyField(ProductOrder)
```
Comment (ben-d-marks) [inline bangazon/bangazon_ui/models/order_model.py:20]: This is unnecessary, and is fixed in PR `Multiple item Fix`

File: bangazon/bangazon_ui/templates/bangazon_ui/order_detail_view.html [hunk 2]
```diff
@@ -1,5 +1,19 @@
-{% extends 'bangazon_ui/index.html'%}
-{% block content %}
-"Hello Are you there?"
+{% extends 'bangazon_ui/index.html'%} {% block content %}
+<h2> ' This is the janky stand in for Order View Template' </h2> {% for item in product_list %}
```
Comment (ben-d-marks) [inline bangazon/bangazon_ui/templates/bangazon_ui/order_detail_view.html]: This should be deleted

File: bangazon/bangazon_ui/templates/bangazon_ui/order_detail_view.html [hunk 3]
```diff
@@ -1,5 +1,19 @@
...
+<h4> {{item.nam}}</h4>
+<br></br>
+<h4> {{item.price}}</h4> {%endfor %}
+</ul>
```
Comment (ben-d-marks) [inline bangazon/bangazon_ui/templates/bangazon_ui/order_detail_view.html]: There is no opening `<ul>` tag

File: bangazon/bangazon_ui/templates/bangazon_ui/order_detail_view.html [hunk 4]
```diff
@@ -1,5 +1,19 @@
...
+<form action="{%url 'bangazon_ui:order_detail'%}" method="post">
+    {% csrf_token %}
+    <select name="payment">
+        {% for payment in payment_types %}
+        <option value="{{payment.pk}}" selected="selected"> Payment {{payment.payment_type}}: {{payment.account_number}}</option>
+        {% endfor %}
+    </select>
```
Comment (ben-d-marks) [inline bangazon/bangazon_ui/templates/bangazon_ui/order_detail_view.html:15]: There's no place for the total here.....Maybe a order detail view problem too?

File: bangazon/bangazon_ui/views/payment_type_view.py [hunk 5]
```diff
@@ -32,13 +32,4 @@ def post(self, request):
...
-
-
-
-
-
-
-
-
-
+        return HttpResponseRedirect(redirect_to='/order_detail_view')
```
Comment (ben-d-marks) [inline bangazon/bangazon_ui/views/payment_type_view.py]: This redirect should go to `/order_detail` per the urls.py

**Review by ben-d-marks — CHANGES_REQUESTED** [review]

File: bangazon/bangazon_ui/templates/bangazon_ui/product_list.html [hunk 6]
```diff
@@ -3,7 +3,7 @@
 
 {% if product_list %}
     <ul>
-    {% for product in product_list %}
+    {% for paym in product_list %}
```
Comment (ben-d-marks) [inline bangazon/bangazon_ui/templates/bangazon_ui/product_list.html]: This breaks a view :(

**Review by ben-d-marks — CHANGES_REQUESTED** [review]

File: bangazon/bangazon_ui/models/order_model.py (hunk 1 above)
Comment (shawn-mit) [reply bangazon/bangazon_ui/models/order_model.py:20]: why is this pushing up i didn't use git add .   - strange

File: bangazon/bangazon_ui/templates/bangazon_ui/order_detail_view.html (hunk 2 above)
Comment (shawn-mit) [reply bangazon/bangazon_ui/templates/bangazon_ui/order_detail_view.html]: yea i agree - this was during the testing

File: bangazon/bangazon_ui/templates/bangazon_ui/order_detail_view.html (hunk 3 above)
Comment (shawn-mit) [reply bangazon/bangazon_ui/templates/bangazon_ui/order_detail_view.html]: we aren't using an unordered list , i don't think it necessary.  why would we want this ul?

File: bangazon/bangazon_ui/templates/bangazon_ui/order_detail_view.html (hunk 4 above)
Comment (shawn-mit) [reply bangazon/bangazon_ui/templates/bangazon_ui/order_detail_view.html:15]: i'll check that out

File: bangazon/bangazon_ui/templates/bangazon_ui/product_list.html (hunk 6 above)
Comment (shawn-mit) [reply bangazon/bangazon_ui/templates/bangazon_ui/product_list.html]: i'll check it . i must've started in the wrong file and then switched over to the correct one for payment in payment_types

**Review by ben-d-marks — COMMENTED** [review]
looks good 👍

**Review by ben-d-marks — APPROVED** [review]

**Review by danirenae — APPROVED** [review]
looks good!

**Comment by danirenae** [comment]
👍 🚀

**Pull request merged**